app/models/Assets.scala

```diff
@@ -18,22 +18,33 @@ package models
 
 import java.time.LocalDate
 
+import play.api.libs.functional.syntax._
 import play.api.libs.json._
 
 sealed trait AssetType
 
-case class Assets(monetary: Option[List[AssetMonetaryAmount]],
-                  propertyOrLand: Option[List[PropertyLandType]],
-                  shares: Option[List[SharesType]],
-                  business: Option[List[BusinessAssetType]],
-                  partnerShip: Option[List[PartnershipType]],
-                  other: Option[List[OtherAssetType]],
-                  nonEEABusiness: Option[List[NonEeaBusinessType]])
+case class Assets(monetary: List[AssetMonetaryAmount],
+                  propertyOrLand: List[PropertyLandType],
+                  shares: List[SharesType],
+                  business: List[BusinessAssetType],
+                  partnerShip: List[PartnershipType],
+                  other: List[OtherAssetType],
+                  nonEEABusiness: List[NonEeaBusinessType])
+
 
 object Assets {
-  implicit val assetsFormat: Format[Assets] = Json.format[Assets]
+  implicit val reads: Reads[Assets] =
+    ((__ \ "assets" \ "monetary").readWithDefault[List[AssetMonetaryAmount]](Nil)
+      and (__ \ "assets" \ "propertyOrLand").readWithDefault[List[PropertyLandType]](Nil)
+      and (__ \ "assets" \ "shares").readWithDefault[List[SharesType]](Nil)
+      and (__ \ "assets" \ "business").readWithDefault[List[BusinessAssetType]](Nil)
+      and (__ \ "assets" \ "partnerShip").readWithDefault[List[PartnershipType]](Nil)
+      and (__ \ "assets" \ "other").readWithDefault[List[OtherAssetType]](Nil)
+      and (__ \ "assets" \ "nonEEABusiness").readWithDefault[List[NonEeaBusinessType]](Nil)
+      ).apply(Assets.apply _)
```

Comment:
I'm wondering if these reads need the `\ "assets" \`? I presume they're applied when getting the assets from the backend but unless I'm mistaken that picks the data from the json at the `assets` level

Reply:
They are needed. I had to add the assets part after doing a manual test and finding that none of the data from trusts was being picked up